working_with_hl7/target/build

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Let's remove target folder from git ignore and update this PR by removing target files

Reply:
Updated in https://github.com/ballerina-guides/healthcare-samples/commit/bded8478e21ac73dc2a93c39487e9999ebf23416